src/modules/cmd_duration.rs

```diff
@@ -101,22 +101,27 @@ fn undistract_me<'a, 'b>(
 ) -> Module<'a> {
     use ansi_term::{unstyle, ANSIStrings};
     use notify_rust::{Notification, Timeout};
+    use std::convert::TryFrom;
 
     if config.show_notifications && config.min_time_to_notify as u128 <= elapsed {
         let body = format!(
             "Command execution {}",
             unstyle(&ANSIStrings(&module.ansi_strings()))
         );
 
-        let mut notification = Notification::new();
-        notification
-            .summary("Command finished")
-            .body(&body)
-            .icon("utilities-terminal")
-            .timeout(Timeout::Milliseconds(750));
-
-        if let Err(err) = notification.show() {
-            log::trace!("Cannot show notification: {}", err);
+        if let Ok(display_time) = u32::try_from(config.notify_display_time) {
```

Comment:
Would it be better to try to parse the timeout and if it fails log an error/warning and then fall back to the default of 750?

Reply:
Depends, not showing the notification at all might prompt the user to go looking for issues sooner.